### ISSUE TITLE
Doesn't work for me in Arch

### ISSUE THREAD

**Issue by Praron** [sboli/twmn#65]
Hello. Thanks for app, it looks very good, but don't work for me.
Installed on Arch from AUR(PKGBUILD not changed), I start twmnd, execute something like "twmnc -c 'Hello world' ", twmnd print "content - Hello world", but nothing happens more, no notification.

Thanks in advice.

**Comment by ixjlyons** [comment]
What are the contents of your `twmn.conf`? Also, make sure another notification daemon isn't running.

**Comment by Praron** [comment]
twmn.conf just generated by twmnd, nothing changed
`
[gui]
absolute_position=
always_on_top=true
background_color=#000000
bounce=true
bounce_duration=500
font=Sans
font_size=13
font_variant=medium
foreground_color=#999999
height=18
in_animation=38
in_animation_duration=1000
opacity=100
out_animation=13
out_animation_duration=1000
position=top_right
screen=

[icons]
critical_icon=
info_icon=
warning_icon=

[main]
activate_command=
duration=3000
enable_shortcuts=true
host=127.0.0.1
port=9797
sound_command=
`
I have no more notification daemons, system is a sort of fresh. Only thing, that makes notification too is a GNotifier extension for Firefox, but as I know it is not a full-featured notification daemon(it is not separated from Firefox process or something like this) and twmn not works even with closed Firefox.

**Comment by ixjlyons** [comment]
Interesting, I'm also on Arch using [twmn-git](https://aur.archlinux.org/packages/twmn-git/) and it works fine for me with that config.

For the record, I realized another notification daemon shouldn't be an issue if you're using `twmnc` to do the messaging -- it communicates with `twmd` through a separate UDP socket instead of D-Bus. Obviously `twmnd` is receiving the UDP messages, since they're being printed out in the terminal.

The only other information I can think of that might help would be your window manager and/or desktop environment.

**Comment by Praron** [comment]
Oh, sorry, I forget about this. I use i3, in config nothing, that can break twmn(I think), no DE installed. A-a-and good news, it works ._.
I don't know what is changed, but it works for now, sorry for anxiety.

**Comment by serussell** [comment]
God, I hate these mysterious self-fixing issues.

I'm having the same issue, but different set-up.  My system isn't fresh, and I've been using `i3` and `twmn` for ... months? Years? It just stopped working, but I think in my case it has something to do with changing my monitor layout.  I have a `position=bottom_right` directive, but no value for `screen=`.  It was working until I undocked, with the message showing up in the right place on the right-most monitor.  After I undocked, the messages stopped appearing.  I restarted `twmnd` and ran it from the command line, and like OP I get the `content` messages when using `twmnc`, but no desktop notification.  `xrandr` correctly reports that there's only one connected monitor, and `i3` certainly thinks there's only one (I can see all my desktops), but I'm guessing that `twmnd` for some reason still thinks there's a monitor further right and that's where it's showing the notifications.  :man_shrugging: 

Is it possible that `twmnd` is getting stuck in a multi-monitor mode, even across restarts of the daemon?
